.codeclimate.yml

```diff
@@ -0,0 +1,4 @@
+plugins:
+  eslint:
+    enabled: true
+    channel: "eslint-7"
```

Comment:
Aren't we using eslint 6?

![image](https://user-images.githubusercontent.com/37511135/100788602-0c84d200-33f4-11eb-93f3-c2c88f755064.png)

Reply:
Solved